### PR TITLE
chore(pulsar): port #14110 and #14291 to e5.8.4

### PR DESCRIPTION
**Pull request by thalesmg**
Port of #14110 and #14291 to e5.8.4

Release version: e5.8.4

## Summary

- Add `telemetry` support to pulsar driver and action.
- Fixed handling of `Redirect` `LookupType` responses in pulsar producer driver.

### CODE REVIEW

**Review by zmstone — APPROVED** [review]
please add changelog for this

**Comment by thalesmg** [comment]
> please add changelog for this

:checkered_flag: 

https://github.com/emqx/emqx/pull/14316/commits/1ecaadf5be2d0d36722f1342913d5dc98baa063b

File: apps/emqx_bridge_pulsar/src/emqx_bridge_pulsar_connector.erl [hunk 1]
```diff
@@ -229,33 +244,97 @@ on_query(_InstanceId, {ChannelId, Message}, State) ->
...
+on_pulsar_ack(ReplyFnAndArgs, Result) ->
+    emqx_resource:apply_reply_fun(ReplyFnAndArgs, Result).
+
+%%-------------------------------------------------------------------------------------
+%% `telemetry' API
+%%-------------------------------------------------------------------------------------
+
+%% we *must* match the bridge id in the event metadata with that in
+%% the handler config; otherwise, multiple pulsar producer bridges will
+%% install multiple handlers to the same pulsar events, multiplying the
```
Comment (savonarola) [inline apps/emqx_bridge_pulsar/src/emqx_bridge_pulsar_connector.erl:291]: Nit: ... event counts?

**Review by savonarola — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-5340)

**Pull request merged**